### DISCUSSION TITLE
Cannot Access Web UI

### DISCUSSION THREAD

**Discussion by kgoode517** [Security-Onion-Solutions/securityonion#6626] [Unsupported Versions] [answered]
Hey guys,
I finally successfully installed Security Onion on virtualbox. I had to choose Redhat 64bit linux when installing as there was no Centos linux option like it says in the instructions. From googling it looks like Centos is lumped in with Redhat. I then tried and failed many times to install the onion only to eventually discover that when following the instructions and setting a static ip my vm had no internet access. I switched to a fresh install with dhcp assignment and viloa I finally got a successful install and internet access. I then ran sudo so-status to verify that all of my services are running.  Next I run sudo so-allow and added my ip addresses for access to the web admin page. No dice all I get is a time out. Checking my firewalls and isp I don’t see anything blocking. I finally just add individual ips to so-allow which seems to have worked for my Imac but nothing else. By work I mean I can ping out from the onion to my Imac but not vice versa. Trying to access the web page still just times out. Next I read the FAQ and tried sudo salt-call state.highstate to see if that had any errors. No luck there either. Now I’m trying a tcp dump/ wireshark any idea of where to go from here? Also is there no gui for the onion? It seems to be the terminal and scripts only.

**Comment by tomjones1977** [comment, marked as answer]
It's quite likely that your VM is set up in NAT mode, which means you can connect out but not connect in.  You can either change the networking mode to bridged (which I would recommend as DHCP on your manager interface can cause problems if the IP changes), or you can set up a port forward from your host to the guest.

**Reply by kgoode517** [reply]
Yes sir that was definitely the case. Starting from scratch with a new vm in bridged mode took care of everything. Thank you so much for your help! I didn’t even consider trying to change the network interface but it makes total sense.
